VisualPinball.Unity/Documentation~/creators-guide/tutorials/realistic-backglass/3-import-into-unity.md

```diff
@@ -0,0 +1,46 @@
+---
+uid: tutorial_backglass_3
+title: Realistic backglass - Import Into Unity
+description: How setup your backglass in Unity
+---
+
+# Import Into Unity
+
+## Step 1: Import the FBX
+
+Open your scene in Unity and navigate to where your table stores its Materials, Meshes and textures.  Drag and drop your backglass mesh into "Meshes" and your images into "Textures"
+
+> [!note]
+> We recommend storing models at `Assets/<Table Name>/Models`, and texture at `Assets/<Table Name>/Textures`. In this example the Tropic Fun backglass is being created in another table for demonstrative purposes.
+
+![Navigate to Table Folders](BringMeshAndpngsIntoUnity.jpg)
+
+## Step 2: Add a Mesh Filter
+
+In the Hierarchy window add a new element by right clicking and choosing "create empty". In this case we named it "Backglass".  Right click in the "Inspector" window and search for "Mesh Filter" and add this to the inspector window.  With the Mesh Filter highlighted search for the Blender backglass mesh that you made.  Grab the "cube" mesh from this and drag-drop it into the "mesh" slot in the mesh filter.   If the cube cannot be moved, select the top level (in this case TropicFun BG), right click and select "unpack".
```

Comment:
>  Right click in the "Inspector" window and search for "Mesh Filter" and add this to the inspector window

It's left click in the inspector window, no? I would put something like:

> In the inspector window, click on *Add Component* and find *Mesh Filter*. Locate your exported FBX in the *Project* window, expand it, and drag the *Cube* mesh into the mesh filter's *Mesh* slot.

> If the cube cannot be moved

How did this happen?

Reply:
Yep it should be click "add component".  Sometimes when I've tried to grab parts of a pre-fab I get an error and must unpack the prefab first.  I think the cube with slot icon is a prefab?